### PR TITLE
feat: Add tensor scatter and add function

### LINKED ISSUE #26264: tensor_scatter_nd_add

### PR DESCRIPTION
**Pull request by ayobamiakomolafe**
# Adds sparse updates to an existing tensor according to indices

##26264

Close #26264

## Checklist

- [x] Did you add a function?
- [x] Did you add the tests?
- [ ] Did you run your tests and are your tests passing?
- [ ] Did pre-commit not fail on any check?
- [x] Did you follow the steps we provided?

### CODE REVIEW

**Comment by ayobamiakomolafe** [comment]
Would appreciate if someone can let me know what to do to fix this Failed tests.

File: ivy/functional/frontends/tensorflow/general_functions.py [hunk 1]
```diff
@@ -586,7 +586,12 @@ def strided_slice(
     return ret
 
 
-@with_unsupported_dtypes({"2.14.0 and below": ("uint16",)}, "tensorflow")
+@to_ivy_arrays_and_back
+def tensor_scatter_nd_add(tensor, indices, updates, name=None):
+    return ivy.tensor_scatter_nd_add(tensor, indices, updates)
```
Comment (zhumakhan) [inline ivy/functional/frontends/tensorflow/general_functions.py:591]: It seems ivy functional does not have `tensor_scatter_nd_add` implemented. You can add it or even better if you just use existing functions like `ivy.scatter_nd` and `ivy.add` to implement this

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_general_functions.py [hunk 2]
```diff
@@ -2036,6 +2036,38 @@ def test_tensorflow_strided_slice(
         raise e
 
 
+# tensor_scatter_nd_add
+@handle_frontend_test(
+    fn_tree="tensorflow.tensor_scatter_nd_add",
+    tensor=helpers.array_values(
+        dtype=helpers.get_dtypes("integer"), shape=(8,), min_value=2, max_value=49
```
Comment (zhumakhan) [inline ivy_tests/test_ivy/test_frontends/test_tensorflow/test_general_functions.py:2043]: Please check for all numeric types: `dtype=helpers.get_dtypes("numeric")`

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_general_functions.py [hunk 3]
```diff
@@ -2036,6 +2036,38 @@ def test_tensorflow_strided_slice(
...
+        shape=(4,),
+        min_value=9,
+        max_value=12,
+    ),
+)
+def test_tensorflow_tensor_scatter_nd_add(
+    *, tensor, indices, updates, frontend, test_flags, fn_tree, on_device, backend_fw
+):
+    helpers.test_frontend_function(
+        input_dtypes=["int32"],
```
Comment (zhumakhan) [inline ivy_tests/test_ivy/test_frontends/test_tensorflow/test_general_functions.py:2059]: please also generate dtype like in `test_tensorflow_tile` for example.

**Review by zhumakhan — CHANGES_REQUESTED** [review]

**Comment by Ishticode** [comment]
closing this as it is a duplicate of https://github.com/unifyai/ivy/pull/26361

**Pull request closed without merging**